web/src/TriggerModeToggle.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react"
+import styled from "styled-components"
+import { ReactComponent as AutoSvg } from "./assets/svg/auto.svg"
+import {
+  AnimDuration,
+  Color,
+  ColorAlpha,
+  mixinResetButtonStyle,
+  SizeUnit,
+} from "./style-helpers"
+import { TriggerMode } from "./types"
+
+let TriggerModeToggleStyle = styled.button`
+  width: ${SizeUnit(1)};
+  height: ${SizeUnit(1)};
+
+  ${mixinResetButtonStyle}
+
+  .fillStd {
+    fill: ${Color.blue};
+    transition: fill ${AnimDuration.short} linear;
+  }
+  .strokeStd {
+    stroke: ${Color.blue};
+    opacity: ${ColorAlpha.almostOpaque};
+    transition: stroke ${AnimDuration.short} linear;
+  }
+  .autoSvgToggleOn {
```

Comment:
maybe call this autoSvg-toggleOn, to signify that this is the "toggleOn" component of the auto svg?

Reply:
Good naming suggestion, thanks!